components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/util/SAMLSSOUtil.java

```diff
@@ -1179,7 +1179,9 @@ public static boolean validateDeflateSignature(String queryString, String issuer
             return samlHTTPRedirectSignatureValidator.validateSignature(queryString, issuer, certificate);
 
         } catch (SecurityException e) {
-            log.error("Error validating deflate signature", e);
+            if (log.isDebugEnabled()) {
+                log.debug("Error validating deflate signature", e);
```

Comment:
Can we include some context information like issuer etc.?

Reply:
Fixed https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/307/commits/c26bd6e382113980690594a16ac03992c7531edf